test/suite/test_hs25.py

```diff
@@ -0,0 +1,45 @@
+import wttest
```

Comment:
The test is missing the file header. There is also another test test_hs24.py also missed a file header. Can you fix that also as part of this ticket?